src/plugins/analysis/exploit_mitigations/view/exploit_mitigations.html

```diff
@@ -2,15 +2,27 @@
 
 {% block analysis_result_details %}
 
-    {% for key in analysis_result|sort %}
-        {% if key not in ['summary', 'plugin_version', 'system_version', 'analysis_date', 'tags', 'skipped'] %}
-            <tr>
-                <td>{{ key }}</td>
-                <td class="table-{% if (analysis_result[key] == 'enabled' or analysis_result[key] == 'fully enabled') %}success{% elif analysis_result[key] == 'disabled' %}danger{% else %}warning{% endif %}">
-                    {{ analysis_result[key] | nice_generic | safe }}
-                </td>
-            </tr>
+    {% for key, value in analysis_result.items() | sort %}
+        {% if value in [True, 'enabled', 'fully enabled'] %}
+            {% set bs_class = 'success' %}
+        {% elif value in [False, 'disabled'] %}
+            {% set bs_class = 'danger' %}
+        {% else %}
+            {% set bs_class = 'warning' %}
         {% endif %}
+
+        <tr>
+            <td>{{ key }}</td>
+            <td class="table-{{ bs_class }}">
+                {% if value == True %}
+                    enabled
+                {% elif value == False %}
+                    disabled
+                {% else %}
+                    {{ value }}
+                {% endif %}
+            </td>
```

Comment:
I don't see the benefit of hiding the non-boolean information from relro and pie here. Couldn't we just add this?

Reply:
I'm not sure I'm following. The output should be "enabled" if it `True`, "disabled" if it is `False` or the value (a string in this case) otherwise. What are we hiding?

Reply:
Nevermind, I missed the `else`.